diff -ru rhash-Rhash-1.1/rhash/rhash.py rhash-Rhash-1.1-fixed/rhash/rhash.py
--- rhash-Rhash-1.1/rhash/rhash.py	2022-07-09 22:11:12.000000000 +0200
+++ rhash-Rhash-1.1-fixed/rhash/rhash.py	2023-01-02 22:45:52.484590282 +0100
@@ -80,17 +80,7 @@
 )
 
 # initialization
-if sys.platform == "win32":
-    _LIBNAME = "librhash.dll"
-elif sys.platform == "darwin":
-    _LIBNAME = "librhash.0.dylib"
-elif sys.platform == "cygwin":
-    _LIBNAME = "cygrhash.dll"
-elif sys.platform == "msys":
-    _LIBNAME = "msys-rhash.dll"
-else:
-    _LIBNAME = "librhash.so.0"
-_LIBRHASH = CDLL(_LIBNAME)
+_LIBRHASH = CDLL("@librhash@")
 _LIBRHASH.rhash_library_init()
 
 # function prototypes
